src/test/java/com/rarchives/ripme/tst/ripper/rippers/DeviantartRipperTest.java

```
@@ -50,11 +48,6 @@ public void testGetGID() throws IOException, URISyntaxException {
     @Disabled("Broken ripper")
     public void testGetGalleryIDAndUsername() throws IOException, URISyntaxException {
         URL url = new URI("https://www.deviantart.com/airgee/gallery/").toURL();
-        DeviantartRipper ripper = new DeviantartRipper(url);
```

I don't like making changes to a broken test, because the intention of the test is to not be broken. The reason for commenting out parts of this test were to make the test run pass, and that's what resulted in the unused variables warnings you're probably trying to fix here. DeviantArtRipper is broken for now and we need to fix it, not hide the errors further by removing more code from the test.

Please drop these changes from this PR.

---

Surely would  drop it